hack/lib/lint.sh

```diff
@@ -33,4 +33,10 @@ kubeedge::lint::check() {
     golangci-lint run
     gofmt -l -w staging
     set -o pipefail
+
+    # check vendor licenses
+    ./hack/verify-vendor-licenses.sh || {
+      echo "Some vendor licenses do not exist, please call hack/update-vendor-licenses.sh to update them"
+      return 1
+    }
```

Comment:
lgtm, and does the licenses verify belong to lint check? Or add it to the `make verify`.

Reply:
Let's move it to `make verify`.